package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "dotenv -e .env.dev -- vite",
```

Comment:
Do we need this?
because Vite uses `dotenv` under the hood
you can simply load your environment variables by running `vite`
is there any reason that I missed it?

Reply:
the thing is that being explicit is usually not a bad thing and IMO relying on vite to load that out of the box is not a good approach (it has some specific rule on naming env files as I know and we might not be following that )
besides, we might (actually going to) need other envs for different environment (e.g cypress)

Reply:
With this approach, we still rely on `vite` to load our envs
but I agree to be explicit and we're going to use it in other places

Reply:
Nice discussion! :)